examples/lsm303agr/main.go

```diff
@@ -28,22 +34,24 @@ func main() {
 		})
 	*/
 
-	if !sensor.Connected() {
-		println("LSM303AGR/MAG not connected!")
-		return
-	}
-
 	for {
+
+		if con, err := sensor.Connected(); !con || err != nil {
+			println("LSM303AGR/MAG not connected!")
+			time.Sleep(time.Second)
+			continue
+		}
+
 		// accel_x, accel_y, accel_z := sensor.ReadAcceleration()
```

Comment:
Are these supposed to be commented out here?

Reply:
Have no idea, not my code, I've just made it work with refactorings.
I suppose, since it's an example, this shows how to read raw acceleration, instead of Pitch and Roll.